src/shared/components/adminTabs/FlashRound/RoundAnswer.tsx

```diff
@@ -41,7 +41,7 @@ const RoundAnswer = (props: any) => {
                         variant="standard"
                         value={points}
                         onChange={(e) => {
-                            setPoints(+e.target.value);
+                            setPoints(e.target.value);
```

Comment:
Here, instead of using the `+`, let's check if the value is a number and ignore the input if it is not.